pkg/injector/init_container.go

```diff
@@ -23,6 +25,8 @@ func getInitContainerSpec(containerName string, cfg configurator.Configurator, o
 					"NET_ADMIN",
 				},
 			},
+			RunAsNonRoot: &runInitContainerAsNonRoot,
+			RunAsUser:    &initContainerUserId,
```

Comment:
`import "k8s.io/utils/pointer"`
and
```suggestion
			RunAsNonRoot: pointer.BoolPtr(false),
			RunAsUser:    pointer.Int64Ptr(0),
```